framework/src/meshmodifiers/SidesetAroundSubdomainUpdater.C

```diff
@@ -25,12 +25,18 @@ SidesetAroundSubdomainUpdater::validParams()
                                               "Subdomains that own the boundary");
   params.addParam<std::vector<SubdomainName>>("outer_subdomains",
                                               "Subdomains on the outside of the boundary");
+  params.addParam<BoundaryName>("mask_side",
+                                "If specified, only add sides where this sideset exists.");
   params.addParam<bool>("assign_outer_surface_sides",
                         true,
                         "Assign sides of elements im `inner_subdomains` that have no neighbor.");
   params.addRequiredParam<BoundaryName>("update_sideset_name",
                                         "The name of the sideset to be updated. If the boundary "
                                         "does not exist it will be added to the system.");
+  params.addParam<BoundaryID>("update_sideset_id",
```

Comment:
either rename these two to boundary_id/name

OR 
add the corresponding ndoeset parameters